README.md

```diff
@@ -1,11 +1,28 @@
 Friendica Social Communications Server
 ======================================
 
-
 Welcome to the free social web.
 
+Friendica is a decentralised communications platform that integrates social communication. Our platform links to independent social projects and corporate services.
+
+Our mission is to free friends, family and colleagues from data-harvesting corporations; we aim for social communication to be free and open, while flowing between any provider as easily as email does.
+
+Friendica connects you effortlessly to a federated communications network of several thousand servers, with more than half a million user registrations. You can directly connect to anyone on [Friendica]( https://friendi.ca), [Mastodon](https://joinmastodon.org/), [Diaspora](https://diasporafoundation.org/), [GnuSocial](https://gnu.io/social/), [Pleroma](https://pleroma.social/), or [Hubzilla](https://hubzilla.org/), regardless where each user profile is hosted.
+
+With Friendica, you can also fully interact with anyone on Twitter, post on Facebook and receive any content on Tumblr, Wordpress or RSS. Friendica allows you to integrate most things on the web via a range of plugins such as ITTT, Buffer; you will be able to easily control your own data as you decide.
+
+Join today and [get your Friendica profile!](https://dir.friendica.social/servers 'Join Friendica today!')
+
 
-Friendica is a communications platform for integrated social communications utilising decentralised communications and linkage to several indie social projects - as well as popular mainstream providers.
 
-Our mission is to free our friends and families from the clutches of data-harvesting corporations, and pave the way to a future where social communications are free and open and flow between alternate providers as easily as email does today.
+### Friendica Screenshots
 
+![Alt text](/images/screenshots/friendica-frio-mobile-profle-1.png?raw=true "Frio theme on a mobile")
```

Comment:
Shouldn't `Alt text` be replaced by actual text alternative?

Reply:
It should, yes.